### ISSUE TITLE
Mixed Case with Greek Letters

### ISSUE THREAD

**Issue by zoenglinghou** [deszoeke/LaTeXSusbstitutionsPlist#1]
I have been looking for this a long time, thank you for realizing this awesome project!

However, I found that the Greek letter substitutions only give results of capital letters, i.e. `\alpha` returns Α, `beta` returns Β.

Could it be a glitch of macOS, or font issues?

**Comment by deszoeke** [comment]
Yes, I found the huge list of symbols created havoc with autocorrect, which is really designed to guess spelling mistakes. For example, it doesn’t require the backslash to start guessing these substitutions. For better performance, I suggest winnowing down the list of more than 1000 symbols to just the 100 or so Greek and common math symbols you’ll probably use. That helped for me.

I experienced the same issue with capital Greek letters. I think the solution was to to remove capital Greek letter substitutions that look just like capital Roman letters. For my simple purposes, having different characters that look the same is likely  to lead to errors.

Simon
from my mobile

**Comment by zoenglinghou** [comment]
@deszoeke 
I found a partial solution to this.

Since I am using [alfred](https://www.alfredapp.com/), I converted your `.plist` into an Alfred Snippet Collection: [plist2Alfred](https://github.com/CarlosEvo/plist2Alfred).
